modular_zubbers/modules/jobs/job_types/blacksmith.dm

```diff
@@ -0,0 +1,61 @@
+/datum/job/blacksmith
+	title = JOB_BLACKSMITH
+	description = "Smith wares, Sell them."
+	department_head = list(JOB_HEAD_OF_PERSONNEL)
+	faction = FACTION_STATION
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = SUPERVISOR_HOP
+	config_tag = "BLACKSMITH"
+	exp_granted_type = EXP_TYPE_CREW
+
+	outfit = /datum/outfit/job/blacksmith
+	plasmaman_outfit = /datum/outfit/plasmaman/curator //Not sure how this works. Leaving it alone.
+
+	paycheck = PAYCHECK_CREW
+	paycheck_department = ACCOUNT_SRV
+
+	display_order = JOB_DISPLAY_ORDER_BLACKSMITH
+	departments_list = list(
+		/datum/job_department/service,
+		)
+
+	mail_goodies = list(
+		/obj/item/book/random = 44,
+		/obj/item/book/manual/random = 5,
+		/obj/item/book/granter/action/spell/blind/wgw = 1,
```

Comment:
I looked this up and while it's hilarious it's probably not really appropriate for Blacksmiths. Some fun gimmick vaguely related to their job, or something genuinely useful seems appropriate, but this is merely a suggestion, feel free not to add anything!

Reply:
as for this, this is a placeholder from curator, this will be changed later